packages/thirdweb/src/chainsaw/endpoints/getBlock.ts

```diff
@@ -0,0 +1,41 @@
+import type { GetBlockReturnType } from "viem";
+import type { ThirdwebClient } from "../../client/client.js";
+import { getClientFetch } from "../../utils/fetch.js";
+import { formatChainsawBlock } from "../formatter.js";
+import type { Block, ChainsawResponse } from "../types.d.ts";
+import { getBlockEndpoint } from "../urls.js";
+
+export type GetBlockParams = {
+  client: ThirdwebClient;
+  blockNumber: number;
+  chainId: number;
+};
+
+/**
```

Comment:
Could you add more detail to the jsdocs? They'll be the primary docs for these

Reply:
Added more docs in [e915687](https://github.com/thirdweb-dev/js/pull/3741/commits/e9156876ed106429684c41e6933caef0659ecb00)